DTS-STN/Alpha-Site
# French translation

# Description

[French content changes](https://trello.com/c/qLzDgPHt/420-french-content-changes)

I updated the French content.

## Acceptance Criteria

The French content changes mentioned in the Trello card should be modified.

## Test Instructions

1. Explorez nos projets changes to Explorer nos projets (changes to the menu, breadcrumb and page title)
2. On the Home page, the button label on FR Apprenez-en davantage sur les laboratoires de Service Canada changes to **En savoir plus sur les laboratoires de Service Canada**
3 . "Date modifieé" should be "Date de modification"
4. Check the linguistic review in the trello card and check to se if the changes were made

## Checklist


## Product and Sprint Backlog

[Trello](https://trello.com/b/ZqWtJSyh/alpha-site-board)
